pyemma/coordinates/data/_base/iterable.py

```diff
@@ -351,7 +351,7 @@ def _n_chunks(self):
         return min(n1, n2)
 
     def __len__(self):
-        return min(self._it.trajectory_lengths(), self._it_lagged.trajectory_lengths())
+        return int(min(self._it.trajectory_lengths(), self._it_lagged.trajectory_lengths()))
```

Comment:
Yeah I was a bit confused because you seem to do min on two numpy arrays which would not work if they have length more than one.
Changed it now in any case.